cli/cli

```diff
@@ -233,6 +234,9 @@ else
     elif docker exec ${VM_CID} /bin/bash -c "test -f /scripts/nodes.sh" ; then
       docker exec ${VM_CID} /bin/bash -c "ssh.sh sudo /bin/bash < /scripts/nodes.sh"
     fi
+    if docker exec ${VM_CID} /bin/bash -c "test -f /scripts/logging.sh" ; then
```

Comment:
no need we use `gocli` to run containers, so I think this part of `cli` will be removed soon